resources/dicts/thoughts/alive/apprentice.json

```diff
@@ -236,28 +348,221 @@
+            "Hopes that training under r_c will help {PRONOUN/m_c/object} be as confident as {PRONOUN/m_c/poss} mentor",
+            "Runs off crying after getting a stern lecture by r_c",
+            "Wishes r_c could be a bit more understanding",
+            "Feels like r_c doesn't like {PRONOUN/m_c/object}",
+            "On a hard day, m_c is surprised when r_c gives {PRONOUN/m_c/object} the day off",
+            "Gets a lecture on being more confident when {PRONOUN/m_c/subject} complain about never succeeding in a move",
+            "Knows r_c is trying {PRONOUN/r_c/poss} best to help {PRONOUN/m_c/object}",
+            "Is found by r_c crying in {PRONOUN/m_c/poss} nest, and is comforted",
+            "m_c is surprised when {PRONOUN/m_c/poss} mentor brings {PRONOUN/m_c/object} {PRONOUN/m_c/poss} favorite prey",
+            "Gets lessons on posturing confidently from r_c",
+            "Gets scared of {PRONOUN/m_c/poss} mentor after r_c snaps at {PRONOUN/m_c/object}",
+            "r_c apologizes for snapping at m_c the other day"
+
+        ],
+        "main_trait_constraint": [
+            "insecure",
+            "nervous"
+        ],
+        "random_trait_constraint": [
+            "troublesome",
+            "fierce",
+            "bloodthirsty",
+            "cold",
+            "strict",
+            "ambitious",
+            "vengeful",
+            "arrogant",
+            "grumpy",
+            "rebellious"
+
+        ],
+        "random_living_status": [
+            "living"
+        ],
+        "relationship_constraint": [
+            "app/mentor"
+        ]
+    },
+    {
+        "id": "insecure_nervous_warrior_apprentice_to_outgoing_mentor",
+        "thoughts": [
+            "Wishes {PRONOUN/m_c/poss} mentor didn't encourage {PRONOUN/m_c/object} to speak with others so much",
+            "Hides behind r_c during the gathering",
+            "Wonders how r_c can speak with others so easily",
+            "Tries to do some suggestions r_c gave {PRONOUN/m_c/object} to be able to be more outgoing",
+            "Hides away from {PRONOUN/m_c/poss} mentor when {PRONOUN/m_c/subject} {VERB/m_c/learn/learns} {PRONOUN/m_c/poss} mentor wants to take {PRONOUN/m_c/object} to the far edges of the territory today"
+
+        ],
+        "main_trait_constraint": [
+            "insecure",
+            "nervous"
+        ],
+        "random_trait_constraint": [
+            "bold",
+            "daring",
+            "righteous",
+            "adventurous",
+            "flamboyant",
+            "arrogant"
+
+        ],
+        "random_living_status": [
+            "living"
+        ],
+        "relationship_constraint": [
+            "app/mentor"
+        ]
+    },
+    {
+        "id": "insecure_nervous_warrior_apprentice_to_withdrawn_mentor",
+        "thoughts": [
+            "m_c looks at r_c... an awkward silence hangs between them",
```

Comment:
```suggestion
            "Looks at r_c as an awkward silence hangs between them",
```

Reply:
This removes the feeling of awkwardness, the eclipses are used as a stylistic choice to create a pause in the readers mind, and give the impression of silence ^^

Reply:
I just don't think it reads well? 🤷 It reads as "this sentence is awkward" not "this situation is awkward", but that's possibly just personal preference lol

Reply:
That might be, we can let scribble decide on this one. I'd rather not have it changed, but if it *needs* to be changed then I will.

Reply:
Nah, if you prefer it then disregard this suggestion ^-^
(I can't mark as resolved apparently, so you'll have to. Consider this resolved from my end tho!)